### ISSUE TITLE
Totally bad data

### ISSUE THREAD

**Issue by hkristof03** [ranaroussi/yfinance#1052]
I just came across this library and performed some initial queries but what I see is a bit far from reality. 

```
aapl = yf.Ticker("AAPL")
bs = aapl.balance_sheet

print(f"{bs.transpose()['Common Stock'][::-1][-1]:,}")
```

57,365,000,000.0

Checking [outstanding shares are more like ~16,701,272,000-](https://finance.yahoo.com/quote/AAPL/financials?p=AAPL)

For MSFT it is also wrong.

The cash flow statement contains "net income", which shouldn't be an element of it.

**Comment by niceguy4** [comment]
@hkristof03 

What's the issue? It appears to be reporting correctly. 

When display the full frame for 'bs' in your example, you get the same 57mil. The important item is the date, '2021-09-25'. Here is [Apple's balance sheet from that timeframe](https://www.apple.com/newsroom/pdfs/FY21_Q4_Consolidated_Financial_Statements.pdf). The numbers in your example match Apple's official balance sheet statement for that timeframe.

**Comment by hkristof03** [comment]
Yes that would be better probably, at least according to my knowledge. Never heard that Common stock and Shareholder's equity are used equivalently. The former for me indicates the number of outstanding common shares, while the latter is what is left over after the total liabilities is subtracted from total assets.

Similarly there is no net income on the cash flow statement. Free cash flow and net income are not the same.

**Comment by ValueRaider** [comment]
Check out this PR, it might solve your problem: https://github.com/ranaroussi/yfinance/pull/776

**Comment by ValueRaider** [comment]
Correct financials are now available in pre-release version 0.2.0rc4.
